README.md

```diff
@@ -1,16 +1,98 @@
+
+## Unacceptable Behavior
+
+The following behaviors are considered harassment and are unacceptable within our community:
+
+* Violence, threats of violence, or violent language directed against another person.
+* Sexist, racist, homophobic, transphobic, ableist, or otherwise discriminatory jokes and language.
+* Posting or displaying sexually explicit or violent material.
+* Posting or threatening to post other people’s personally identifying information ("doxing").
+* Personal insults, particularly those related to gender, sexual orientation, race, age, religion/nonreligion, or disability.
+* Inappropriate photography or recording.
+* Unwelcome sexual attention. This includes, sexualized comments or jokes.
+* Deliberate intimidation, stalking or following (online or in person).
+* Advocating for, or encouraging, any of the above behavior.
+* Sustained disruption of on-topic Slack channels.
+
+## Consequences of Unacceptable Behavior
+
+Unacceptable behavior from any community member, including sponsors and those with decision-making authority, will not be tolerated.
+
+Anyone asked to stop unacceptable behavior is expected to comply immediately.
+
+If a community member engages in unacceptable behavior, the community organizers may take any action they deem appropriate, up to and including a temporary ban or permanent expulsion from the community without warning.
+
+## Reporting Guidelines
+
+If you are subject to or witness unacceptable behavior, or have any other concerns, please notify a NewHavenIO board member as soon as possible. You'll likely get a faster response on Slack than by e-mail.
+
+Henry Quinn (@henry)
+Jenn Nolte (@Jenn Nolte she/her)
+Josh Levinson (@jlevnhv)
+Steve Voils (@steve)
+Tom Reznick (@tom)
+Ben Midyette (@ben)
+
+All board email addresses are FirstName@newhaven.io.
+
+## Addressing Grievances
+
+If you feel you have been falsely or unfairly accused of violating this Code of Conduct, you should notify NewHavenIO admins with a concise description of your grievance. Your grievance will be handled in accordance with our existing governing policies.
+
+## Scope
+
+We expect all community participants (contributors, paid or otherwise; sponsors; and other guests) to abide by this Code of Conduct in all community venues–online and in-person–as well as in all one-on-one communications pertaining to community business.
+
+This code of conduct and its related procedures also applies to unacceptable behavior occurring outside the scope of community activities when such behavior has the potential to adversely affect the safety and well-being of community members.
+
+## Contact info
+
+Feel free to contact any of the NewHavenIO board members at any time. You'll likely get a faster response on Slack than by e-mail.
+
+Henry Quinn (@henry)
+Jenn Nolte (@Jenn Nolte she/her)
+Josh Levinson (@jlevnhv)
+Steve Voils (@steve)
+Tom Reznick (@tom)
+Ben Mindyette (@ben)
```

Comment:
Typo here (Midyette)

Reply:
Good catch!

Reply:
Although it has a ring to it... bmindy? lol